tests/conftest.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any, Optional
+
+import pytest
+
+from mkosi.config import OutputFormat
+from mkosi.distributions import Distribution, detect_distribution
+
+
+def pytest_addoption(parser: Any) -> None:
+    def parse_integration_distribution(s: str) -> set[Optional[Distribution]]:
+        if s == "hostonly":
+            return {detect_distribution()[0]}
+        return {Distribution(d) for d in s.split(",")}
+
+    def parse_integration_output(s: str) -> set[OutputFormat]:
+        if s == "all":
+            return {f for f in OutputFormat if f != OutputFormat.none}
+        return {OutputFormat(o) for o in s.split(",")}
+
+    parser.addoption(
+        "-D",
```

Comment:
Why not a lower case d?

Reply:
pytest doesn't seem to use all that many short options, but all of the ones it has are lowercase. I thought it safer to steer clear of lowercase options.